### PR TITLE
ipn/ipnstate: update field docs on PeerStatus.

### PR DESCRIPTION
**Pull request by maisem**
Signed-off-by: Maisem Ali <maisem@tailscale.com>

### CODE REVIEW

**Review by DentonGentry — APPROVED** [review]

File: ipn/ipnstate/ipnstate.go [hunk 1]
```diff
@@ -183,13 +183,18 @@ type PeerStatusLite struct {
...
-	HostName     string // HostInfo's Hostname (not a DNS name or necessarily unique)
-	DNSName      string
-	OS           string // HostInfo.OS
-	UserID       tailcfg.UserID
-	TailscaleIPs []netip.Addr // Tailscale IP(s) assigned to this node
+	ID        tailcfg.StableNodeID
+	PublicKey key.NodePublic
+	HostName  string // HostInfo's Hostname (not a DNS name or necessarily unique)
+
+	// DNSName is the peers FQDN, it ends with a dot.
```
Comment (bradfitz) [inline ipn/ipnstate/ipnstate.go:190]: Peer's. Semicolon or period.

File: ipn/ipnstate/ipnstate.go [hunk 2]
```diff
@@ -223,7 +228,14 @@ type PeerStatus struct {
 	// change.
 	Active bool
 
-	PeerAPIURL   []string
+	// PeerAPIURL are the URLs of the node's PeerAPI servers.
+	PeerAPIURL []string
+
+	// Capabilities are capabilities that the node has.
```
Comment (bradfitz) [inline ipn/ipnstate/ipnstate.go:234]: Reference some constants too?

**Review by bradfitz — APPROVED** [review]
LGTM in general

File: ipn/ipnstate/ipnstate.go (hunk 2 above)
Comment (maisem) [reply ipn/ipnstate/ipnstate.go:234]: I did

**Pull request merged**